scripts/image_checker.rb

```diff
@@ -0,0 +1,49 @@
+# Checks image constraints and fails the travis build if they are not
+class ImageChecker
```

Comment:
ImageChacker is the gem img_checker which was included. We developed it during the last year's competition